docs/plugin_matrix.rst

```diff
@@ -39,8 +39,12 @@ btv                     btv.bg               Yes   No    Requires login, and geo
 canalplus               mycanal.fr           No    Yes   Streams may be geo-restricted to France.
 cdnbg                   - tv.bnt.bg          Yes   No    Streams may be geo-restricted to Bulgaria.
                         - bgonair.bg
-                        - kanal3.bg
                         - bitelevision.com
+                        - bloombergtv.bg
```

Comment:
Did you mean to add this? I don't see a test for this specific domain.

Reply:
Yes this URL was added in some other PR #1168
I updated this PR with a new test and removed another domain.